docker-compose.override.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.8'
+
+services:
+
+  nginx:
+    image: nginx:alpine
```

Comment:
I'd suggest using the same images of prod, then customise whatever you like inside of them using the volumes.

Reply:
totally agreed